documentation/api/query/v4/nodes.markdown

```diff
@@ -59,6 +68,19 @@ The below fields are allowed as filter criteria and are returned in all response
     Note that nodes which are missing a fact referenced by a `not` query will match
     the query.
 
+### Subquery Relationships
+
+Here is a list of relationships that can be queried using implicit subqueries. For
+more information consult the documentation for [subqueries].
+
+* [`factsets`][factsets]: Factsets received from this node.
+* [`reports`][reports]: Reports received from this node.
+* [`catalogs`][catalogs]: Catalogs received for this node.
+* [`facts`][facts]: Fact names and values received from this node.
+* [`fact_contents`][fact-contents]: Factset paths and values received from this node.
+* [`events`][events]: Report events triggered from this node.
+* [`edges`][edges]: Catalog edges received for this node.
+* [`resources`][resources]: Catalog resources received for this node.
```

Comment:
I find the "this node" phrasing a bit confusing, since the subquery is actually constraining the list of nodes returned via some arbitrary field on the target table. I'm not sure what the right thing to say is

```
[resources][resources]: Constrain result to nodes matching a resource query
```
kind of clunky -- any thoughts?